content/en/providers/getting-started/operations_start_guide/_index.md

```diff
@@ -0,0 +1,281 @@
+institution or national borders, who, by reason of their common membership and
+in sharing a common goal, are given authority to use a set of resources. Each VO
+member signs the VO Acceptable Usage Policy (AUP) (during registration) which is
+the policy describing the goals of the VO thereby defining the expected and
+acceptable use of the resources by the users of the VO. User documentation can
+be found [in the users section](../../../users/) and
+[in the EGI Wiki](https://wiki.egi.eu/wiki/User_Documentation).
+
+#### VO manager
+
+An individual responsible for the management of the membership registry of the
+VO ensuring its accuracy and integrity.
+
+## Joining operations
+
+In order to join any of the organisational groups in your NGI, you will need to
+go through the following steps in order:
+
+### Authentication
+
+The authentication in EGI infrastructure works either with X509 personal
+certificates or through federated identities using [Check-in](../../check-in).
+
+#### Obtaining a X509 personal certificate
+
+If you do not already have a X509 personal certificate
+[the EUGridPMA worlmap](https://www.eugridpma.org/members/worldmap/) provides a
+map of all certification authorities according to the country (or NGI). Select
+your country on the map to find out who is your local CA. Follow the procedure
+of your local CA to request a certificate. When you have received your
+certificate, install it into your web browser.
+
+If case of setting up a new Resource Centre please request Host certificates.
+
+EUGridPMA provides
+[a web page allowing to test your certificate](https://www.eugridpma.org/your-identity/).
+Please use this resource and contact your CA if your certificate does not work.
+
+#### Create a federated identity: registration in EGI Check-in
+
+As soon as you try to access an EGI service with your federated identity, you
+will be requested to register an account in EGI Check-in if not existing yet.
+The [Check-in sign up guide](../../../users/check-in/signup/) explains how to
+sign up for an EGI account. If you already own an account, you will be simply
+asked to login through EGI Check-in.
+
+### Joining dteam VO
+
+It is recommended to join the [dteam VO](https://wiki.egi.eu/wiki/Dteam_vo) at
+the [dteam Registration](https://voms2.hellasgrid.gr:8443/voms/dteam/) page. You
+should request group membership for `/dteam` and `/dteam/YOUR_NGI`. The dteam
+group manager will then be notified and review your application.
+
+### Requesting GOCDB access
+
+- Read
+  [Input System User Documentation](https://wiki.egi.eu/wiki/GOCDB/Input_System_User_Documentation)
+  first.
+- Go to the [Configuration Database](https://goc.egi.eu/) and follow
+  [the instruction](https://wiki.egi.eu/wiki/GOCDB/Input_System_User_Documentation#Users_and_roles)
+
+All new members **need to notify their NGI operations manager** about their role
+request, as the Configuration Database currently does not send any notification
+about pending requests.
```

Comment:
@paolini78 is it still the case? If yes, should we create a requirement for this?

Reply:
it happens quite often that people misses this notifications, so I would mention that the requestors need to notify separately the NGI managers, so the latter can also verify who made the request.

Reply:
I've updated it.